test/teiserver/battle/balance_lib_internal_test.exs

```diff
@@ -0,0 +1,59 @@
+defmodule Teiserver.Battle.BalanceLibInternalTest do
+  @moduledoc """
+  Can run all balance tests via
+  mix test --only balance_test
+  """
+  use ExUnit.Case
+  import Mock
+  @moduletag :balance_test
```

Comment:
I'm not convinced by this tag. Test tags should be used for logical grouping only. Like `tag: cluster` when you have several tests spanning multiple files testing one feature.
If you only want to test one file, you can always run `mix test path/to/the/file`

Reply:
The other balance tests are tagged already though. So it's hitting multiple files.